Weekly_Work/Week1/Code_changes-git/Metrics_Root_class.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import json
+import datetime
+import matplotlib.pyplot as plt
+import pandas as pd
```

Comment:
Please, produce two versions of the root class: one to work with Pandas, and the other one to work with lists of directories, as produced by Perceval (well, after importing the JSON documents that Perceval produces).

Reply:
Will do!

Reply:
Thx!